Kernel/KBO.cpp

```diff
@@ -955,14 +879,9 @@ bool KBO::isGreater(AppliedTerm lhs, AppliedTerm rhs) const
   return isGreaterOrEq(lhs,rhs)==GREATER;
 }
 
-bool KBO::isGreater(TermList lhs, TermList rhs, const SubstApplicator* applicator, OrderingComparatorUP& comparator) const
+OrderingComparatorUP KBO::createComparator(TermList lhs, TermList rhs) const
```

Comment:
This could be worth putting in the header file depending on how hot it is.

Reply:
I tried doing it but some incomprehensible error came up about some cyclic inclusion. I don't think it's that hot though.